application-analytics-ui/src/main/resources/Analytics/Code/AnalyticsTranslations.xml

```diff
@@ -73,7 +73,17 @@ analytics.rowEvolution.modal.month.label=Month
 analytics.rowEvolution.modal.year.label=Year
 analytics.rowEvolution.modal.field.pageViews=Page views
 analytics.rowEvolution.modal.field.uniqueViews=Unique views
-analytics.rowEvolution.modal.field.avgTime=Avg. Time spent
+analytics.rowEvolution.modal.field.avgTime=Time spent  on page
```

Comment:
Space added by mistake. 
So this is not done as an average for now, as I remember we discussed (because Matomo does not send the avg, but the total time spent). Could the key also be updated to reflect the displayed value?